src/test/java/org/opentripplanner/updater/alerts/AlertsUpdateHandlerTest.java

```diff
@@ -31,15 +29,13 @@
 import org.opentripplanner.routing.services.TransitAlertService;
 import org.opentripplanner.util.TranslatedString;
 
-@RunWith(MockitoJUnitRunner.class)
 public class AlertsUpdateHandlerTest {
 
   private AlertsUpdateHandler handler;
 
-  @Spy
-  private FakeTransitAlertService service;
+  private final FakeTransitAlertService service = Mockito.spy(FakeTransitAlertService.class);
```

Comment:
That simple :-), I search for a solution and didn´t find it ... Thank you.

Reply:
Yeah, I wonder what the benefit of using the the specialised runner was. It's more code AND more magic...